<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="92650" started-at="2018-11-15T17:09:03Z" finished-at="2018-11-15T17:10:35Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="92650" started-at="2018-11-15T17:09:03Z" finished-at="2018-11-15T17:10:35Z">
      <class name="com.MagicBricks.scenarios.FkartLoginScenario">
        <test-method status="PASS" signature="setupSuite(org.testng.ITestContext)[pri:0, instance:com.MagicBricks.scenarios.FkartLoginScenario@3eb7fc54]" name="setupSuite" is-config="true" duration-ms="30" started-at="2018-11-15T22:39:03Z" finished-at="2018-11-15T22:39:03Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@5454d35e]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupSuite -->
        <test-method status="PASS" signature="openBrowser()[pri:0, instance:com.MagicBricks.scenarios.FkartLoginScenario@3eb7fc54]" name="openBrowser" is-config="true" duration-ms="10826" started-at="2018-11-15T22:39:03Z" finished-at="2018-11-15T22:39:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method status="PASS" signature="reportHeader(java.lang.reflect.Method, org.testng.ITestContext)[pri:0, instance:com.MagicBricks.scenarios.FkartLoginScenario@3eb7fc54]" name="reportHeader" is-config="true" duration-ms="68" started-at="2018-11-15T22:39:13Z" finished-at="2018-11-15T22:39:13Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.MagicBricks.scenarios.FkartLoginScenario.TC_1_Login() throws java.lang.Throwable]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@5454d35e]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reportHeader -->
        <test-method status="FAIL" signature="TC_1_Login()[pri:0, instance:com.MagicBricks.scenarios.FkartLoginScenario@3eb7fc54]" name="TC_1_Login" duration-ms="80674" started-at="2018-11-15T22:39:13Z" finished-at="2018-11-15T22:40:34Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Unable to click on Login expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Unable to click on Login expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.ctaf.accelerators.ActionEngine.click(ActionEngine.java:89)
	at com.MagicBricks.workFlows.MagicBricksBusinessFuctions.FkartLogin(MagicBricksBusinessFuctions.java:419)
	at com.MagicBricks.scenarios.FkartLoginScenario.TC_1_Login(FkartLoginScenario.java:25)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_1_Login -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.MagicBricks.scenarios.FkartLoginScenario@3eb7fc54]" name="tearDown" is-config="true" duration-ms="1066" started-at="2018-11-15T22:40:34Z" finished-at="2018-11-15T22:40:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestContext)[pri:0, instance:com.MagicBricks.scenarios.FkartLoginScenario@3eb7fc54]" name="tearDown" is-config="true" duration-ms="15" started-at="2018-11-15T22:40:35Z" finished-at="2018-11-15T22:40:35Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@5454d35e]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.MagicBricks.scenarios.FkartLoginScenario -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
